packages/controls/src/controls/group/__snapshots__/group.test.ts.snap

```diff
@@ -175,18 +181,19 @@ ShapeV2Definition {
       },
     },
   },
+  "dataType": "group::v1",
```

Comment:
Since this snapshot is an object dump of the `Group` definition instance, it reflects the fact that the definition now has a runtime `dataType` (which isn't serialized anywhere else). More on this below.